osbuild/api.py

```diff
@@ -193,6 +202,29 @@ def _cleanup(self):
             os.close(self._output_pipe)
             self._output_pipe = None
 
+def exception(e, path="/run/osbuild/api/osbuild"):
+    """Send exception to osbuild"""
+    traceback.print_exception(type(e), e, e.__traceback__, file=sys.stderr)
+    exc = {"type": str(type(e)),
+           "value": str(e),
+           "traceback": str(e.__traceback__)
+          }
```

Comment:
nitpick: the formatting is a bit off here.

Reply:
you could also combine it with the dict creation below (`msg`).

Reply:
noted, thanks!